src/downshift.js

```diff
@@ -248,7 +248,7 @@ class Downshift extends Component {
 
   scrollHighlightedItemIntoView() {
     /* istanbul ignore else (react-native) */
-    if (preval`module.exports = process.env.BUILD_REACT_NATIVE !== 'true'`) {
+    if (!isReactNative) {
```

Comment:
Out of curiosity... With this new macro, can we remove these Istanbul ignore comments?

Reply:
Maybe, i kinda dont know how those comments work **exactly**, not sure how to check if they can be removed (like i dont know what kind of outcome should i expect and where)

Reply:
Try to remove the comment, then run `npm run test -- --coverage` and see if we still get a 100% coverage report.

Reply:
Rats! Istanbul uses a babel plugin to instrument the code for coverage and I've found it's been a little tough to make it play nicely with other babel plugins at times :-(

Reply:
https://github.com/paypal/downshift/pull/542#issuecomment-412681801

it seems to work ok now, CI reports TS task failing, will have to investigate this in following days